tests/drivers/imu/test/support/adis_generic_tests.c

```diff
@@ -0,0 +1,4064 @@
+/***************************************************************************//**
```

Comment:
why not name this file test_adis.c ? this way naming would be consistent with the other test files

Reply:
you are right, thank you!